ocaml/testsuite/tests/typing-layouts/basics_alpha.ml

```diff
@@ -1337,7 +1347,7 @@ let q () =
   assert false
 
 [%%expect{|
-val ( let* ) : 'a -> ('b -> t_float64) -> unit = <fun>
+val ( let* ) : 'a ('b : any). 'a -> ('b -> t_float64) -> unit = <fun>
```

Comment:
Please add comments toward the end of the file for the newly added tests, just to keep the test numbers lined up. No need to copy the actual tests.